lib/queries.rb

```diff
@@ -14,77 +14,92 @@ def self.default
         "most-common-likers": {
             "id": -1,
             "name": "Most Common Likers",
-            "description": "Which users like particular other users the most?"
+            "description": "Which users like particular other users the most?",
+            "group_ids": []
```

Comment:
if it is just going to be blank everywhere is there any point including it?

Reply:
Yeah, agreed.